index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
한국어 웹사이트인 경우 `lang="ko"`로 수정해주세요.

```suggestion
<html lang="ko">
```

Reply:
넵!!